test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -93,3 +97,390 @@ def test_exclusive_job_not_suspended(self):
+                VNODE, {'resources_available.vntype': 'cray_compute'})
+            vlist = vnl["resources_available.vntype=cray_compute"]
+        except Exception:
+            sys.exc_clear()
+            self.logger.info("There are no cray_compute vnodes present.")
+        numvnl = len(vlist)
+
+        now = time.time()
+        resv_start = now + 180
+        resv_dur = 7200
+        d = self.submit_resv(resv_start, numvnl, resv_dur)
+        self.assertTrue(d)
+
+        j = Job(TEST_USER, {ATTR_l + '.select': '1:ncpus=1',
+                            ATTR_l + '.min_walltime': '00:02:00',
+                            ATTR_l + '.max_walltime': '02:00:00'})
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid)
+        self.server.expect(
+            JOB, {ATTR_l + '.walltime': (LE, '00:03:00')}, id=jid)
+        self.server.expect(
+            JOB, {ATTR_l + '.walltime': (GE, '00:02:00')}, id=jid)
+
+        # The sleep below will leave less than 2 minutes window for jid
+        # after j2id is deleted.  The min_walltime of jid can't be
+        # satisfied and jid will stay in S state.
+        time.sleep(65)
+
+        j2 = Job(
+            TEST_USER, {
+                ATTR_l + '.select': '%d:ncpus=1:vntype=cray_compute' % numvnl,
+                ATTR_l + '.walltime': '00:01:00',
+                ATTR_l + '.place': 'scatter',
+                ATTR_q: 'highp'})
+        j2id = self.server.submit(j2)
+
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=j2id)
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=jid)
+
+        self.server.delete(j2id)
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+        self.server.expect(SERVER, {'server_state': 'Active'})
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=jid)
+        self.server.cleanup_jobs()
+
+    def test_multi_express(self):
+        """
+        Test of multiple express queues of different priorities.
+        See that jobs from the higher express queues preempt jobs
+        from lower express queues.  Also see when express jobs finish
+        (or are deleted), suspended jobs restart.
+        Make sure loadLimit is set to 4 on the server node:
+        # apmgr config loadLimit 4
+        """
+
+        _t = ('\"express_queue, normal_jobs, server_softlimits,' +
+              ' queue_softlimits\"')
+        a = {'preempt_prio': _t}
+        self.scheduler.set_sched_config(a)
+
+        try:
+            self.server.manager(MGR_CMD_DELETE, QUEUE, id="expressq")
+        except:
+            pass
+        a = {'queue_type': 'e',
+             'started': 'True',
+             'enabled': 'True',
+             'Priority': 150}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, "expressq")
+
+        try:
+            self.server.manager(MGR_CMD_DELETE, QUEUE, id="expressq2")
+        except:
+            pass
+        a = {'queue_type': 'e',
```

Comment:
You don't have to create dict a everytime, just update priority like this -  a['Priority']=160